### DISCUSSION TITLE
recipes/desserts/1-bowl-vegan-gluten-free-carrot-cake/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#324] [Comments]
# recipes/desserts/1-bowl-vegan-gluten-free-carrot-cake/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/1-bowl-vegan-gluten-free-carrot-cake/

**Comment by nicholaswilde** [comment]
Convert step 6 to note admonition
